Source/WebCore/dom/CharacterData.h

```diff
@@ -43,6 +43,8 @@ class CharacterData : public Node {
     // Like appendData, but optimized for the parser (e.g., no mutation events).
     void parserAppendData(StringView);
 
+    ContainerNode* parentOrShadowHostNode() const = delete; // Call parentNode() instead.
```

Comment:
I sometimes make the return value `void` and make these private as well.

Reply:
Ok.